.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   rules: {
     "no-param-reassign": 0,
     "import/no-extraneous-dependencies": 0,
-    "react/no-string-refs": 1
+    "react/no-string-refs": 1,
+    "import/extensions": [2, "always", {js: "never"}]
```

Comment:
What about having this rule part of [eslint-config-algolia](https://github.com/algolia/eslint-config-algolia)?

Reply:
It's a first step :) But sure it's gonna end up there if we're happy about it.

Reply:
https://github.com/algolia/eslint-config-algolia/pull/92 we'll remove the rule when it's merged and the config updated.